src/circuit/gadget/ecc.rs

```diff
@@ -114,7 +114,7 @@ pub trait EccInstructions<C: CurveAffine>: Chip<C::Base> {
     ) -> Result<Self::Point, Error>;
 
     /// Performs variable-base scalar multiplication, returning `[scalar] base`.
-    /// Multiplication of the identity [a]𝒪 returns an error.
+    /// Multiplication of the identity [a] 𝒪 returns an error.
```

Comment:
Hmm, this didn't fix the doc check failure. What command is being run for doc checks? None of `cargo doc`, `cargo check`, or `cargo clippy` reproduce this for me.

Reply:
Oh, it runs `cargo doc --document-private-items`.

Reply:
```suggestion
    /// Multiplication of the identity `[a] 𝒪` returns an error.
```